### ISSUE TITLE
Control the order of terrain files when combining terrain in one job

### ISSUE THREAD

**Issue by OmarShehata** [AnalyticalGraphicsInc/cesium-ion-community#259]
Cesium ion allows you to tile raster terrain data, as well as combine it with one or more layers. Currently the only way to do this is to upload one asset, wait for it to finish, then upload the next one and say it should use the first asset as a base layer.

In order to support uploading all assets at once, we need to either support specifying the expected order of the layers if they're uploaded together, or support choosing a base layer as an asset that it is still being processed, and have the new job wait on the previous one before starting.

Right now uploading two raster terrain files will combine them, but the user has no control over which layer gets planed on top. This was brought up in the discussion of a separate issue here: https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues/256#issuecomment-564847631

**Comment by OmarShehata** [comment]
We're migrating the issues in this repository to consolidate questions, Cesium ion feature requests, and bug reports on the [Cesium community forum](https://community.cesium.com/), that way there'll be one place to search for answers. 

If you're interested in this feature, please open a new topic in the community forum!

**Comment by hyunjiklee** [comment]
Spam mail is coming to you right now.  Please take action

나의 iPhone에서 보냄
